app/assets/javascripts/components/articles/PaginatedArticleControls.jsx

```diff
@@ -19,7 +19,8 @@ export const PaginatedArticleControls = ({ showMore, limitReached }) => {
   const filteredArticles = useSelector(getArticlesByTrackedStatus);
   const currentPage = useSelector(state => state.articles.currentPage);
   const pageRange = getPageRange(currentPage, filteredArticles.length);
-  const totalEditedArticles = useSelector(state => state.course.edited_count);
+  const totalEditedArticles = getPageRange(currentPage, filteredArticles.length).split('-')[1];
```

Comment:
I think this has the potential to be confusing, because the variable name in this case seems to be the total number of filtered articles rather than the total of all articles edited. Maybe a change to the variable name will make it more clear?

Also, I'm confused about the implementation here. Is there an easier way to get the right value than splitting the string?

Reply:
Okay i will change the variable name and as for the implementation, I have to check i am not sure if there is any other way to get the total number of filtered articles.

Reply:
I have changed the implementation too.